yoeunes/notify-symfony-toastr/0.1/config/packages/notify_toastr.yaml

```diff
@@ -0,0 +1,8 @@
+notify_toastr:
```

Comment:
It looks like you added the default config here. 
The bundle would behave exactly the same if this file was removed, right?